site/docs/v1/tech/apis/_webauthn-register-complete-request-body.adoc

```diff
@@ -0,0 +1,40 @@
+==== Request Body
+
+The [field]#credential# in the request body contains data returned by the WebAuthn client API call. The [field]#credential.response.attestationObject# and [field]#credential.response.clientDataJSON# fields are returned as ``ArrayBuffer``s by the WebAuthn client API and must be converted to base64url-encoded strings for the FusionAuth API request.
```

Comment:
Everything that is part of `credential` on this request comes from the authenticator response in the browser and shouldn't be changed other than converting fields from `ArrayBuffer`s to base64url-encoded strings.

Maybe I should make that more explicit somewhere?